src/arrayf/arrayf.ts

```diff
@@ -68,4 +68,45 @@ export class arrayf {
         return maxValue;
     }
 
+    /**
+     * Determines if the value in each index of each list matches the
+     * corresponding value in that same index in each other list.
+     * @param lists
+     */
+    static areArrayValuesIdentical<T>(lists: T[][]): boolean {
```

Comment:
I'm not sure of the performance but I wonder if is the same thing and faster?

return JSON.stringify(a)==JSON.stringify(b);

Reply:
It looks like JSON.stringify is very slow by comparison: https://jsbench.me/z5kitbq0tr/1